packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -402,14 +420,14 @@ export class UnfurlService {
                             width: gridWidth ?? viewport.width,
                         });
                     }
-                    await page.on('requestfailed', (request) => {
+                    page.on('requestfailed', (request) => {
```

Comment:
why did you remove the await ?  in case of error, will it throw an `uncaught error` that can break the server ?

Reply:
Because the `await` had no effect on this expression, 

<img width="744" alt="image" src="https://github.com/lightdash/lightdash/assets/7611706/9186c2da-778b-4b64-a4f4-94655bd120f0">

The error would be caught anyway by the `try/catch` wrap I believe